internal/core/item_tree.rs

```diff
@@ -335,8 +335,8 @@ impl ItemRc {
         !clip.is_empty()
             && clip.max.x >= geometry.min.x
             && clip.max.y >= geometry.min.y
-            && clip.min.x <= geometry.max.x
-            && clip.min.y <= geometry.max.y
+            && clip.min.x < geometry.max.x
+            && clip.min.y < geometry.max.y
```

Comment:
This seems incorrect to me. For example `max.x` is `x + width`, so it's basically one pixel *outside* the rectangle. So `<` seems correct to me, while `<=` would make the rectangle go from `x` to `x + width` inclusive.

@ogoffart is my interpretation correct?

Reply:
For me there still was a bug in this function but on the lines above that I didn't touch previously.